src/main/java/com/example/jwt/faker/UserFaker.java

```diff
@@ -0,0 +1,39 @@
+package com.example.jwt.faker;
+
+import com.example.jwt.entity.User;
+import com.example.jwt.repository.UserRepository;
+import com.github.javafaker.Faker;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class UserFaker implements FakerInterface<User> {
+
+    private final UserRepository userRepository;
+
+    public User create(){
+
+        Faker faker = new Faker();
+
+        return userRepository.save(User.builder()
+                .email(faker.bothify("????##@gmail.com"))
+                .username(faker.bothify("????????#######"))
+                .password(faker.bothify("????????#######"))
```

Comment:
why these data is like bullshit? like `????` or `###`

Reply:
Just check the factory documentation, you will figure it out.